CMakeLists.txt

```diff
@@ -143,7 +143,6 @@ endif(NOT USE_PREBUILT_LLVM)
 #
 set (COMPILE_OPTIONS_TD  opencl_clang_options.td)
 set (COMPILE_OPTIONS_INC opencl_clang_options.inc)
-find_program(LLVM_TABLEGEN_EXE "llvm-tblgen" ${LLVM_TOOLS_BINARY_DIR})
```

Comment:
This change will probably affect building opencl-clang with prebuilt llvm.

Reply:
Synced with @hewj03, we should add USE_PREBUILT_LLVM guard for prebuilt llvm, or there is CMake Error:
LLVM_TABLEGEN_EXE not set.